### ISSUE TITLE
hc sandbox - minor inline documentation mistake

### ISSUE THREAD

**Issue by matthme** [holochain/holochain#1648]
The command as mentioned in the inline documentation [here](https://github.com/holochain/holochain/blob/3266a8c0833dd8bce43b3e3aef38d1c057b94218/crates/hc_sandbox/src/cmds.rs#L33):
```
    /// For example `hc gen -r path/to/my/chains -n 3 -d=first,second,third`
    /// will create three sandboxes with directories named "first", "second", and "third".
```
does not actually seem to work when called like this. Instead it generates only one directory named `/tmp/first,second,third`.

What does work instead is  `hc gen -r path/to/my/chains -n 3 -d first second third`.

Making a wild guess I would say it may work if the `parse(from_os_string)` were added to the structopt macro like [here](https://docs.rs/structopt/latest/structopt/#subcommands).

**Comment by ThetaSinner** [comment]
@matthme are you happy for this to be closed now that #2080 has been merged?

**Comment by matthme** [comment]
yes, looks fine
